src/cli/cmds/sync.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import path from 'path';
+import {program} from 'commander';
+import {
+  getRechunkConfig,
+  getIOSPath,
+  getAndroidPath,
+  doesKeywordExist,
+  updateFile,
+  LOGO,
+} from '../lib';
+
+/**
+ * Defines a command for the "sync" operation using the "commander" library.
+ *
+ * @example
+ * ```bash
+ * yarn rechunk sync
+ * ```
+ *
+ * @remarks
+ * This command is part of a larger program defined using the "commander" library.
+ *
+ * @see {@link https://www.npmjs.com/package/commander | commander} - Command-line framework for Node.js.
+ */
+program
+  .command('sync')
+  .description('sync keys from the ReChunk configuration with native files')
+  .action(async () => {
+    console.log();
+    console.log(LOGO);
+
+    const rc = getRechunkConfig();
+
+    const iosPath = getIOSPath();
+    const infoPlistPath = path.resolve(iosPath, 'example', 'Info.plist');
+
+    if (!fs.existsSync(infoPlistPath)) {
+      throw new Error('[ReChunk]: cannot find Info.plist file.');
+    }
+
+    if (!doesKeywordExist(infoPlistPath, '<key>ReChunkPublicKey</key>')) {
+      throw new Error(
+        '[ReChunk]: cannot find ReChunkPublicKey key in Info.plist file.',
+      );
+    }
```

Comment:
We will not want to throw an error here. For the first use case of a project this key will not exist, this is the ideal case. If the keyword does exist we will just want to warn the user via a logged message that the key exists and it's being overwritten.